components/script/script_runtime.rs

```diff
@@ -473,3 +473,13 @@ unsafe fn set_gc_zeal_options(rt: *mut JSRuntime) {
 #[allow(unsafe_code)]
 #[cfg(not(feature = "debugmozjs"))]
 unsafe fn set_gc_zeal_options(_: *mut JSRuntime) {}
+
+pub struct RawJSContext(*mut JSContext);
```

Comment:
I believe this is a misnomer -- the struct that wraps the `*mut JSContext` is actually the _safe_ JSContext, since we make sure that we don't have a null pointer reference when creating the struct. I would instead rename the JSContext import to `RawJSContext`.

Reply:
thanks, I'll address your advice in next commit.